Userland/Utilities/fortune.cpp

```diff
@@ -151,5 +168,8 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     if (stdout_is_tty)
         outln(); // Tasteful spacing
 
+    if (wait_before_termination)
+        sleep((unsigned int)max(chosen_quote.length() / chars_per_second, minimum_secs_to_wait));
```

Comment:
The cast here should not be C-style. In fact, it is not even required anymore once the type of the constants is corrected.